src/main/java/com/amihaiemil/docker/Containers.java

```diff
@@ -95,10 +95,17 @@ Container create(
      * @see <a href="https://docs.docker.com/engine/api/v1.35/#operation/ContainerList">Docker API Docs</a>
      */
     Containers filter(Map<String, Iterable<String>> filters);
-    
+
     /**
      * Return the Docker engine where these Containers came from.
      * @return Docker.
      */
     Docker docker();
+
+    /**
+     * Get this container.<br><br>
+     * @param containerId of the Container
+     * @return this container.
+     */
+    Container get(final String containerId);
```

Comment:
@jackpan123 Please make a separate PR for this ``get`` method and also write some tests for it.